storage/interface.go

```diff
@@ -145,6 +145,11 @@ type SelectHints struct {
 	Grouping []string // List of label names used in aggregation.
 	By       bool     // Indicate whether it is without or by.
 	Range    int64    // Range vector selector range in milliseconds.
+
+	// TrimDisabled allows to disable trimming matching series chunks based on query Start and End time.
+	// When disabled the returned samples may be outside the queried time range but Select() performances
+	// may be improved.
```

Comment:
The suggestion from `may be` -> `may contain samples` so that it does not mean that all `returned samples` will be outside the range.
```suggestion
	// TrimDisabled allows to disable trimming of matching series chunks based on query Start and End time.
	// When disabled, the returned samples may contain samples outside the queried time range but Select() performances
	// may be improved.
```